lib/v2/screens/receive_screens/receive_enter_data/interactor/receive_enter_data_bloc.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:async/async.dart';
+import 'package:bloc/bloc.dart';
+import 'package:seeds/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/page_commands.dart';
+import 'package:seeds/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_events.dart';
+import 'package:seeds/v2/screens/receive_screens/receive_enter_data/interactor/viewmodels/receive_enter_data_state.dart';
+import 'package:seeds/v2/blocs/rates/viewmodels/rates_state.dart';
+import 'package:seeds/v2/domain-shared/page_state.dart';
+import 'package:seeds/v2/domain-shared/shared_use_cases/get_available_balance_use_case.dart';
+import 'mappers/user_balance_state_mapper.dart';
+
+/// --- BLOC
+class ReceiveEnterDataBloc extends Bloc<ReceiveEnterDataEvents, ReceiveEnterDataState> {
+  ReceiveEnterDataBloc(RatesState rates) : super(ReceiveEnterDataState.initial(rates));
+
+  @override
+  Stream<ReceiveEnterDataState> mapEventToState(ReceiveEnterDataEvents event) async* {
+    if (event is LoadUserBalance) {
+      yield state.copyWith(pageState: PageState.success);
```

Comment:
yield state.copyWith(pageState: PageState.success);
should be 
yield state.copyWith(pageState: PageState.loading);

Reply:
good catch, fix